### DISCUSSION TITLE
package flatpack

### DISCUSSION THREAD

**Discussion by ngiannip** [Keruspe/GPaste#417] [Ideas]
It would be great if gpaste was available as a flatpack package on flathub, to make it easier to have the latest version and installation in general.

**Comment by Keruspe** [comment]
I'm generally fine with the idea, there are just two issues here I think:
- I don't know if we can provide the gnome-shell extension through flatpak 
- I don't have the necessary time for this, nor the use for it, so it will need to be a contribution
